chat/command.go

```diff
@@ -155,7 +155,13 @@ func InitCommands(c *Commands) {
 			}
 
 			oldID := member.ID()
-			member.SetID(SanitizeName(args[0]))
+      newID := SanitizeName(args[0])
+      if newID == oldID {
+        body := "New nickname the same as previous nickname."
+        u.Send(message.NewSystemMsg(body, u))
```

Comment:
I think you can just return an error here, like above. Please try to keep it in the same style as the other errors.

```go
return errors.New("new name is the same as the original")
```